speedTest.js

```diff
@@ -21,6 +21,7 @@ function createProvider(name, url, chainId) {
   const providers = {
     ethereum: createProvider("ethereum", "https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161", 1),
     bsc: createProvider("bsc", "https://bsc-dataseed4.binance.org", 56),
+    candle: createProvider("candle", "https://candlelabs.org", 534),
```

Comment:
please revert changes to this file

Reply:
Is there a reason?

Reply:
My bad! Thank you

Reply:
@samisbakedham still need to revert this change

Reply:
Fixed it